learning_journal_basic/templates/edit.jinja2

```diff
@@ -1,10 +1,11 @@
 {% extends 'index.jinja2' %}
 {% block body %}
-  <form action="" id='edit-article'>
+  <form action="{{ request.route_url('edit_view', id=article.id) }}" method="POST" id="edit-article">
     <fieldset>
       <legend>Edit an article</legend>
-      <input type="text" name="title" placeholder="Article Title" id='title' required>
-      <input type="text" name="body" placeholder="Article Body" id='body' required>
+      <input type="text" name="title" placeholder="Article Title" id='title' value="{{ article.title }}" required>
+      <input type="text" name="body" placeholder="Article Body" id='body' value="{{ article.body }}" required>
```

Comment:
The body for your posts would probably make more sense as a `<textarea>`. Right now it looks as if you're only supposed to write a very small amount as the input for your journal entry. This also applies to your `entry.jinja2` template.